src/components/subscription-choices/subscription-choices.scss

```
@@ -0,0 +1,90 @@
+@use '../../compiled/tokens/scss/breakpoint';
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/a11y';
+@use '../../mixins/theme';
+
+///
+/// Theme properties
+///
+
+// Light theme
+@include theme.props {
+  --theme-color-background-subscription-form: #{color.$base-white};
+}
+
+// Dark theme
+@include theme.props(dark) {
+  --theme-color-background-subscription-form: #{color.$brand-primary};
+}
+
+// Alternate theme
+@include theme.props(alternate) {
+  --theme-color-background-subscription-form: #{color.$base-gray-lighter};
+}
+
+///
+/// Subscription Choices
+///
+
+.c-subscription-choices {
+  position: relative;
+}
+
+.c-subscription-choices__controls-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+  margin: size.$rhythm-default 0;
+
+  .c-button-swap {
+    flex-grow: 1;
+  }
+
+  .c-button-swap__subscribe {
+    display: flex;
+  }
+
+  @media (min-width: breakpoint.$s) {
+    flex-direction: row;
+  }
+}
+
+.c-subscription-choices__control {
+  flex-grow: 1;
+}
+
+///
+/// 1. Ensures the form background color matches the current theme
+///
+
+.c-subscription-choices__form {
+  background-color: var(--theme-color-background-subscription-form); // 1
```

> I was unable to figure out how to set the form background color for the "dark alternate" theme. Any ideas @tylersticka?

You can accomplish this by using the same property as the theme itself:

```suggestion
  background-color: var(--theme-color-background-base); // 1
```

---

Genious! 🎉